### PR TITLE
docs: note on loading dumps

### PR DESCRIPTION
**Pull request by cristianmtr**
- added note on all Searcher Executors about how to import data from dumps

### CODE REVIEW

File: jinahub/indexers/searcher/compound/NumpyFileSearcher/README.md [hunk 1]
```diff
@@ -22,6 +22,25 @@
...
+Since this is a "Searcher"-type Executor, it does not _index_ new data. Rather they are write-once classes, which take as data source a `dump_path`. 
+
+This can be provided in different ways:
+
+- in the YAML definition
+  
+```yaml
+jtype: NumpyFileSearcher
+with:
+	dump_path: /tmp/your_dump_location
```
Comment (maximilianwerk) [inline jinahub/indexers/searcher/compound/NumpyFileSearcher/README.md:36]: spaces instead of tab
```suggestion
    dump_path: /tmp/your_dump_location
```

**Review by maximilianwerk — CHANGES_REQUESTED** [review]
LGTM, except spaces instead of tabs.

File: jinahub/indexers/searcher/compound/NumpyFileSearcher/README.md (hunk 1 above)
Comment (cristianmtr) [reply jinahub/indexers/searcher/compound/NumpyFileSearcher/README.md]: done for all

**Review by maximilianwerk — APPROVED** [review]
LGTM

**Pull request merged**